x-pack/platform/plugins/shared/observability_solution/observability_ai_assistant/server/functions/elasticsearch.ts

```diff
@@ -15,6 +15,7 @@ export function registerElasticsearchFunction({
 }: FunctionRegistrationParameters) {
   functions.registerFunction(
     {
+      strict: false,
```

Comment:
We should use `strict: true` everywhere imo to get [Structured Outputs](https://platform.openai.com/docs/guides/function-calling#structured-outputs) - although I don't know how this behave with other LLMs. Do you @dgieselaar or @pgayvallet ?

Reply:
Ad discussed on slack - I'll take a look at how we could add support for openAI's `strict` function option without messing up the abstraction layer